certified-connectors/Repfabric Lead Loader/apiDefinition.swagger.json

```diff
@@ -0,0 +1,361 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Lead Loader",
+    "description": "This Connector helps automate the process of creating a \"Lead\" for Repfabric's backend system. It takes data from a csv and invokes multiple API calls that can create or get the items needed to automate the Lead creation process.",
+    "contact": {
+      "name": "Andre Roberts",
+      "url": "https://phiquest.com/",
+      "email": "andre@phiquest.io"
+    },
+    "version": "1.0"
+  },
+  "host": "xe8g8pkprc.execute-api.us-east-2.amazonaws.com",
```

Comment:
@andreroberts1119 can you please tell me what environment is this host used for? Is it production or test?

Reply:
@andreroberts1119 we can't certify connectors that have test or staging or non-production host URLs. You need to either have a dynamic host policy template to accept host URL from the users or have a production host URL here.

Reply:
Are we allowed to use a service such as API Gateway from AWS to host our production connection for the certified connector? We are finished with testing, and we would like to save money and use this lambda routine rather than setup our own hosted server for production until we expand on the project.

Reply:
@andreroberts1119 you are allowed to use such service as AWS Cloud, but you can't use testing/staging URL for host. The host URL should be a production one.

Reply:
We shifted our staging work to another URL and converted this URL to be our production URL now.

Reply:
Thank you!